cinema/migrations/0002_actor_cinemahall_genre_movie_actors_movie_genres.py

```diff
@@ -0,0 +1,73 @@
+# Generated by Django 4.1 on 2024-12-08 22:11
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("cinema", "0001_initial"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="Actor",
+            fields=[
+                (
+                    "id",
+                    models.BigAutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                ("first_name", models.CharField(max_length=63)),
+                ("last_name", models.CharField(max_length=63)),
+            ],
+        ),
+        migrations.CreateModel(
+            name="CinemaHall",
+            fields=[
+                (
+                    "id",
+                    models.BigAutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                ("name", models.CharField(max_length=255)),
+                ("rows", models.IntegerField()),
+                ("seats_in_row", models.IntegerField()),
+            ],
+        ),
+        migrations.CreateModel(
+            name="Genre",
+            fields=[
+                (
+                    "id",
+                    models.BigAutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                ("name", models.CharField(max_length=255, unique=True)),
+            ],
+        ),
+        migrations.AddField(
+            model_name="movie",
+            name="actors",
+            field=models.ManyToManyField(related_name="movies", to="cinema.actor"),
+        ),
+        migrations.AddField(
+            model_name="movie",
+            name="genres",
+            field=models.ManyToManyField(
+                default=[], related_name="movies", to="cinema.genre"

```

Comment:
The `default` parameter is not valid for `ManyToManyField`. You should remove `default=[]` from this field definition. `ManyToManyField` does not support a default value because it represents a relationship rather than a single value.